commands/http/multifilereader.go

```diff
@@ -64,13 +64,23 @@ func (mfr *MultiFileReader) Read(buf []byte) (written int, err error) {
 
 		// handle starting a new file part
 		if !mfr.closed {
-			if file.IsDirectory() {
+
+			var contentType string
+			if s, ok := file.(*files.Symlink); ok {
+				mfr.currentFile = s
+
+				// TODO(why): this is a hack. pick a real contentType
+				contentType = "symlink"
```

Comment:
On Tue, Sep 01, 2015 at 06:11:01PM -0700, Jeromy Johnson wrote:
> i'd be fine with `application/symlink`. Googling around doesnt
> really give anything official for symlinks.

Sorry for the long silence, but wouldn't it be easier to just use our
regular file MIME type?  Like I did in #1413 with a71c07a3 [1]?

[1]: https://github.com/ipfs/go-ipfs/commit/a71c07a3d227b0bbf55daaec9d34c80e762f9f78#diff-82091aa927c0b41c64d9e94084ba1118R101